manifest_staging/charts/gatekeeper/templates/gatekeeper-gatekeeper-controller-manager-horizontalpodautoscaler.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: autoscaling/v2beta1
```

Comment:
i think we'll want this to be deployed only if autoscaling.enabled is true

Reply:
done